third-party/cmake/BuildJeMalloc.cmake

```diff
@@ -1,3 +1,7 @@
+if(WIN32)
+  message(STATUS "Building jemalloc in Windows is not supported (skipping)")
```

Comment:
I think you need an `else()` here with all the rest of the code (similarly in the other `skipping` files).